src/block-management/block-holder.js

```diff
@@ -91,7 +92,8 @@ export class BlockHolder extends React.Component<PropsType> {
 	}
 
 	blockHolderFocusedStyle() {
-		return isLandscape() ? styles.blockHolderFocusedLandscape : styles.blockHolderFocusedPortrait;
+		return ( isLandscape() || isIpad ) ?
```

Comment:
Could you take a look at the new commit @hypest ? I have updated the mechanism, it is currently determining the window width and the block width, compares them, and if there are spaces left on right&left then it puts the borders. It is not a detailed approach as before but it will work cross platform.

Reply:
cc @etoledom

Reply:
will merge the new version since it seems like addressing those issues but feel free to comment when you are back @hypest I can open a new PR if needed.